ios/RNVirgilCrypto.m

```diff
@@ -386,6 +390,65 @@ - (NSData*) computeHashFor:(NSData*) data using:(VSMHashAlgorithm) alg
     }
 }
 
+RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(generateGroupSession:(NSString*)groupIdBase64)
+{
+    @autoreleasepool {
+        NSError *err;
+        NSData *sessionId = [self computeHashFor:[groupIdBase64 dataUsingBase64] using:VSMHashAlgorithmSha512];
+        sessionId = [sessionId subdataWithRange:NSMakeRange(0, 32)];
+        VSCFGroupSessionTicket *initialEpochTicket = [VSCFGroupSessionTicket new];
+        [initialEpochTicket setRngWithRng: self.crypto.rng];
+        if ([initialEpochTicket setupTicketAsNewWithSessionId:sessionId error:&err] == NO) {
+            return [ResponseFactory fromError:err];
+        }
+        
+        VSCFGroupSessionMessage *initalEpochMessage = [initialEpochTicket getTicketMessage];
+        NSNumber *epochNumber = @([initalEpochMessage getEpoch]);
+        NSData *data = [initalEpochMessage serialize];
+        return [ResponseFactory fromResult:@{ @"sessionId": [sessionId stringUsingBase64],
+                                              @"currentEpochNumber": epochNumber,
+                                              @"epochMessages": @[[data stringUsingBase64]] }];
+    }
+}
+
+RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(importGroupSession:(NSArray<NSString*>*) epochMessagesBase64)
+{
+    @autoreleasepool {
+        NSError *err;
+        NSMutableArray<VSCFGroupSessionMessage*> *epochMessages = [NSMutableArray arrayWithCapacity:[epochMessagesBase64 count]];
+        for (NSString *epochMessageBase64 in epochMessagesBase64) {
+            VSCFGroupSessionMessage *epochMessage = [VSCFGroupSessionMessage deserializeWithInput:[epochMessageBase64 dataUsingBase64]
+                                                                                            error:&err];
+            if (nil == epochMessage) {
+                return [ResponseFactory fromError:err];
+            }
+            [epochMessages addObject:epochMessage];
+        }
+        NSArray<VSCFGroupSessionMessage*> *sortedEpochMessages = [epochMessages sortedArrayUsingComparator:^NSComparisonResult(VSCFGroupSessionMessage* a, VSCFGroupSessionMessage* b) {
+            uint32_t aEpoch = [a getEpoch];
+            uint32_t bEpoch = [b getEpoch];
+            return aEpoch - bEpoch;
+        }];
+        
+        NSMutableArray<NSString *> *sortedSerializedEpochMessages = [NSMutableArray arrayWithCapacity:[sortedEpochMessages count]];
+        
+        VSCFGroupSession *session = [VSCFGroupSession new];
```

Comment:
[[VSCFGroupSession alloc] init] should be used instead of [VSCFGroupSession new]. That is for all classes